proto/panacea/datapool/v2/query.proto

```diff
@@ -86,4 +91,19 @@ message QueryDataValidationCertificatesResponse {
     (gogoproto.nullable) = false
   ];
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
-}
\ No newline at end of file
+}
+
+// QueryDataPassRedeemReceiptsRequest is the request type for the Query/DataPassRedeemReceipts RPC method.
+message QueryDataPassRedeemReceiptsRequest {
+  uint64 pool_id = 1;
+  string redeemer = 3;
+  cosmos.base.query.v1beta1.PageRequest pagination = 4;
```

Comment:
You can use second argument. It is missed when removing round I think :)
And I wonder why you remove the round.

Reply:
This will be fixed in #322 branch.